MdeModulePkg/Universal/Acpi/AcpiTableDxe/AcpiTable.h

```diff
@@ -126,6 +126,14 @@ typedef struct {
       EFI_ACPI_TABLE_SIGNATURE \
       )
 
+//
+// ACPI HOB produced by silicon initialization code will provide the RSDP structure.
+//
+typedef struct {

```

Comment:
1. The HOB definition should be in a public header file.
2. Only "Rsdp" field is used by AcpiTableDxe code logic and the other two fields are not used at all.